lib/read-stream.js

```diff
@@ -27,32 +28,31 @@ function ReadStream( filename, blockMap, options ) {
     throw new Error( 'Missing block map' )
   }
 
-  /** @type {Object} options */
-  this.options = options || {}
+  options = options || {}
 
   if( filename == null && options.fd == null ) {
     throw new Error( 'Missing filename or file descriptor' )
   }
 
-  this.options.verify = this.options.verify != null ?
-    !!this.options.verify : true
-
-  stream.Readable.call( this, this.options )
+  stream.Readable.call( this, options )
 
   /** @type {Number} File descriptor */
-  this.fd = this.options.fd || null
+  this.fd = options.fd || null
   /** @type {String} File path */
   this.path = filename
   /** @type {String} File open flags */
-  this.flags = this.options.flags || 'r'
+  this.flags = options.flags || 'r'
+  /** @type {Number} File mode */
+  this.mode = options.mode
```

Comment:
What is the point of `mode`?
According to node.js docs for `fs.open()`:

> `mode` sets the file mode (permission and sticky bits), but only if the file was created.

Is there any case where a `ReadStream` will be created with a file that does not exist yet?